doc/getting_started/install.rst

```diff
@@ -25,91 +28,160 @@ The installed python is a full version of python. If your specialized
+    process is automated.  Windows and Mac users may want to install using pip as
+    well to get the version with the latest `changes <https://github.com/reflectometry/refl1d/blob/master/CHANGES.rst>`_.
 
 Installing from source
 ======================
 
-Installing the application from source requires a working python environment.
-See below for operating system specific instructions.
+Requirements
+------------
 
-Our base scientific python environment contains the following packages as
-well as their dependencies:
+    - `Python <https://www.python.org/downloads/>`_ >= 3.10 
+    - git
 
-    - python 3
-    - numpy
-    - scipy
-    - matplotlib
-    - wxpython
+You can either use Python from the `python.org <https://www.python.org/downloads/>_` site, or if you prefer, 
+you can use miniforge. Miniforge is an open version of the `conda` program, and is pre-configured to use the `conda-forge` repository.
+This repo is the most useful for open scientific application development.
 
-Once your environment is in place, you can install directly from PyPI
-using pip::
+You can get an installer for your system (MacOS, Windows or Linux) here: https://github.com/conda-forge/miniforge?tab=readme-ov-file#miniforge3
 
-    pip install refl1d
+Run the installer, then from a terminal window (PowerShell or Terminal in Windows) run::
+
+    conda init 
+
+then close and re-open your terminal.
+
+
+Setup Environment
+-----------------
+
+You can create a new virtual environment for Refl1D, or install it in your base environment.  To create a new environment, run::
+
+    # if using miniforge
+    conda create -n refl1d
+    conda activate refl1d
 
-This will install refl1d, bumps and periodictable.
+    # if using regular python
+    python -m venv refl1d
+    source refl1d/bin/activate
 
-You can run the program by typing:
 
-    python -m refl1d.main
+Installation
+------------
 
-If this fails, then follow the instructions in :ref:`contributing` to install
-from the source archive directly.
+To install the application from source, clone the repository and install the
+dependencies::
 
-Windows
--------
+    git clone https://github.com/reflectometry/refl1d.git
+    cd refl1d
+    pip install .
 
-There are couple of options for setting up a python environment on windows:
+If you want to run the webview, you can install the optional extra::
 
-  - `python.org <https://www.python.org/>`_, and
-  - `Anaconda <https://www.anaconda.com/distribution/>`_.
+    pip install .[webview]
 
-With most pypi packages now bundled with wheels, it is now easy to set up a
-development environment using the official python package.  Similarly,
-anaconda provides binaries for all the refl1d dependencies.
+Additional Configuration - Windows 
+``````````````````````````````````
 
-You will need a C/C++ compiler.  If you already have Microsoft Visual C
+To install Refl1D on Windows, you will need a C/C++ compiler.  If you already have Microsoft Visual C
```

Comment:
Since we switched to numba as the default backend, no additional compiler (beyond the one bundled in numba) is needed for refl1d on any platform.  Any references to compilers in this document can be removed.